### PR TITLE
Msiexec update in sigma rules

### PR DESCRIPTION
**Pull request by schatzimangou**
Update on msiexec sigma rules based on:
https://twitter.com/_st0pp3r_/status/1583914244344799235.
Kindly review and merge if you think it adds value.

### CODE REVIEW

File: rules/windows/process_creation/proc_creation_win_msiexec_install_quiet.yml [hunk 1]
```diff
@@ -9,15 +9,25 @@ date: 2022/01/16
...
+            - ' -i'
+            - ' /package'
+            - ' -package'
+            - ' /a'
+            - ' -a'
+            - ' /j'
+            - ' -j'
+        CommandLine|contains:
             - ' /q'
+            - ' -q'
```
Comment (frack113) [inline rules/windows/process_creation/proc_creation_win_msiexec_install_quiet.yml:30]: Can not have 2 `CommandLine|contains` in the same selection
```yaml
    selection_msi:
        Image|endswith: '\msiexec.exe'
        CommandLine|contains:
            - ' /i'
            - ' -i'
            - ' /package'
            - ' -package'
            - ' /a'
            - ' -a'
            - ' /j'
            - ' -j'
     selection_quiet:
        CommandLine|contains:
            - ' /q'
            - ' -q'
    condition: all of selection_*
```
and
```yaml
date: 2022/01/16
modified: 2022/10/23
```

**Review by schatzimangou — COMMENTED** [review]
Changes in detection and modified fields

File: rules/windows/process_creation/proc_creation_win_msiexec_dll.yml [hunk 2]
```diff
@@ -15,6 +16,8 @@ detection:
         Image|endswith: '\msiexec.exe'
         CommandLine|contains|all:
             - ' /z '
+            - ' -z '
+        CommandLine|contains|all:
```
[the remark this answers is not in the document]
Comment (schatzimangou) [reply rules/windows/process_creation/proc_creation_win_msiexec_dll.yml:21]: Changes performed.

File: rules/windows/process_creation/proc_creation_win_msiexec_execute_dll.yml [hunk 3]
```diff
@@ -10,13 +10,16 @@ modified: 2022/03/28
...
 logsource:
     category: process_creation
     product: windows
 detection:
     selection:
         Image|endswith: '\msiexec.exe'
-        CommandLine|contains: ' /y'
+        CommandLine|contains:
+            - ' /y'
+            - ' -y'
```
[the remark this answers is not in the document]
Comment (schatzimangou) [reply rules/windows/process_creation/proc_creation_win_msiexec_execute_dll.yml:22]: modified date changed

File: rules/windows/process_creation/proc_creation_win_msiexec_install_quiet.yml (hunk 1 above)
Comment (schatzimangou) [reply rules/windows/process_creation/proc_creation_win_msiexec_install_quiet.yml:32]: changes made. sorry about that I am usually more careful.

File: rules/windows/process_creation/proc_creation_win_msiexec_dll.yml [hunk 4]
```diff
@@ -5,16 +5,20 @@ description: Detects MsiExec loading a DLL and calling its DllUnregisterServer f
...
     product: windows
     category: process_creation
 detection:
     selection:
         Image|endswith: '\msiexec.exe'
-        CommandLine|contains|all:
+        CommandLine|contains:
             - ' /z '
+            - ' -z '
+        CommandLine|contains|all:
```
Comment (frack113) [inline rules/windows/process_creation/proc_creation_win_msiexec_dll.yml:21]: You will no pass the test and get a `has the selection (selection/CommandLine|contains|all) with a list of only 1 value in detection`
You have to the 2 `CommandLine|contains` in 2 selection like this
```yaml
    selection_msi:
        Image|endswith: '\msiexec.exe'
        CommandLine|contains:
            - ' /z '
            - ' -z '
    selection_dll:
        CommandLine|contains: '.dll'
    condition: all of selection_*
```
`all of selection_*` will be translate to `selection_msi AND selection_dll`

**Review by frack113 — APPROVED** [review]

**Review by nasbench — APPROVED** [review]

**Pull request merged**